src/TestCase.php

```diff
@@ -41,10 +34,6 @@ public function setUp()
         if (! $this->app) {
             $this->refreshApplication();
         }
-
-        if (! $this->factory) {
-            $this->factory = $this->app->make(Factory::class);
```

Comment:
I rather this go to master, an maintain faker as required for testbench (since everyone should load testbench via required-dev).